app/models/user_bet.rb

```diff
@@ -2,4 +2,10 @@ class UserBet < ApplicationRecord
   belongs_to :user
   belongs_to :bet
   belongs_to :bet_option
+
+  scope :winner_total, lambda { |win_opt|
+    where(bet_option: win_opt)
+      .map(&:amount_bet)
```

Comment:
You generally want to avoid performing ruby calculations in scopes. We use scopes primarily to simplify our interactions with the datable and between models